pom.xml

```diff
@@ -50,7 +50,8 @@
 		<openMRSVersion>2.3.0</openMRSVersion>
 		<openmrsTestutilsVersion>1.3</openmrsTestutilsVersion>
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-		<webservices.restModuleVersion>2.28.0</webservices.restModuleVersion>
+		 <webservices.restModuleVersion>2.28.0</webservices.restModuleVersion> 
+		<!-- <webservices.restModuleVersion>2.17</webservices.restModuleVersion> -->
```

Comment:
@reagan-meant Do not comment out code, just delete it

Reply:
@ssmusoke Okay